src/compiler/interpreter.jl

```diff
@@ -202,15 +223,7 @@ function Core.Compiler.abstract_call_gf_by_type(
     sv::AbsIntState,
     max_methods::Int,
 )
-    ret = @invoke Core.Compiler.abstract_call_gf_by_type(
-        interp::AbstractInterpreter,
-        f::Any,
-        arginfo::ArgInfo,
-        si::StmtInfo,
-        atype::Any,
-        sv::AbsIntState,
-        max_methods::Int,
-    )
+    ret = Core.invoke(Core.Compiler.abstract_call_gf_by_type, Tuple{AbstractInterpreter, Any, ArgInfo, StmtInfo, Any, AbsIntState, Int}, interp, f, arginfo, si, atype, sv, max_methods)
```

Comment:
Why this change?

Reply:
We were spending a huge amount of runtime on that function (which appears to compile as jl_invoke). Tried playing around with to fix to no avail

Reply:
Yeah this change doesn't do anything.